tldw_Server_API/app/core/Moderation/moderation_service.py

```
@@ -609,25 +623,66 @@ def _load_runtime_overrides_file(self) -> None:
+    def _runtime_overrides_payload(self, overrides: dict[str, object]) -> dict[str, object]:
+        payload: dict[str, object] = {}
+        if "pii_enabled" in overrides:
+            payload["pii_enabled"] = bool(overrides.get("pii_enabled"))
+        if "categories_enabled" in overrides:
+            cats = overrides.get("categories_enabled")
+            if isinstance(cats, set):
+                payload["categories_enabled"] = sorted(cats)
+            elif isinstance(cats, list):
+                payload["categories_enabled"] = list(cats)
+            elif isinstance(cats, tuple):
+                payload["categories_enabled"] = list(cats)
```

Ruff (0.15.9)

[warning] Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)